Create package leak_tracker_flutter_testing. (#119)
dart-lang/leak_tracker: pkgs/leak_tracker_testing/README.md

[![pub package](https://img.shields.io/pub/v/leak_tracker_testing.svg)](https://pub.dev/packages/leak_tracker_testing)
[![package publisher](https://img.shields.io/pub/publisher/leak_tracker_testing.svg)](https://pub.dev/packages/leak_tracker_testing/publisher)

Coming soon! See https://github.com/flutter/devtools/issues/3951.

The text below is under construction.

## What is this?

> This is test helpers for [leak_tracker](https://pub.dev/packages/leak_tracker).

Reviewer: Same comment as above; it would be good to answer the question about why this package is separate from `leak_tracker`.

[![pub package](https://img.shields.io/pub/v/leak_tracker_testing.svg)](https://pub.dev/packages/leak_tracker_testing)
[![package publisher](https://img.shields.io/pub/publisher/leak_tracker_testing.svg)](https://pub.dev/packages/leak_tracker_testing/publisher)

Coming soon! See https://github.com/flutter/devtools/issues/3951.

## What is this?

> This is test helpers for [leak_tracker](https://pub.dev/packages/leak_tracker).

Reviewer 2: ```suggestion
This is helper for testing [leak_tracker](https://pub.dev/packages/leak_tracker).
```

They are separated from `leak_tracker` to make sure
testing code is not used in production.

## How to use the helpers?

> To test for leaks with descriptive messages, use `isLeakFree` against a `Leaks instance:

Reviewer: ```suggestion
To test for leaks with descriptive messages, use `isLeakFree` against a `Leaks` instance:
```

> To make your test failing in case of leaks with descriptive message, pass the instance of 'Leaks' to the matcher `isLeakFree`:

Reviewer 2: ```suggestion
To test for leaks with descriptive messages, use `isLeakFree` against a `Leaks instance:
```
